docs/setup/database.md

```diff
@@ -58,6 +58,9 @@ This is done using the [`database migrate`](../usage/cli/database.md#database-mi
 mas-cli database migrate
 ```
 
+This should be done before [syncing your configuration file](./general.md#syncing-the-configuration-file-with-the-database)
+after having made a change to your database configuration.
+
```

Comment:
Actually I just changed it so that migrations automatically run when syncing the config in #2432